README.md

```diff
@@ -199,11 +197,32 @@ file inside your source code repository.
 
       * `web-clustering`: Support for Infinispan-based web session clustering.
 
+      * All layers defined by [wildfly-extras Galleon feature-pack](https://github.com/wildfly-extras/wildfly-datasources-galleon-pack).
```

Comment:
the link text is not correct: wildfly-datasources-feature-pack (this make it explicit that this feature pack deals with datasources)

Reply:
Thanks.